packages/plugin-essentials/sources/commands/set/version.ts

```diff
@@ -86,6 +89,8 @@ export default class SetVersionCommand extends BaseCommand {
     let bundleUrl: string;
     if (this.version === `self`)
       bundleUrl = getBundlePath();
+    else if (this.version.match(/^(http[s]?:)/))
+      bundleUrl = this.version;
```

Comment:
For performance reasons, can you move this check after the checks that check for strict equality? (Those are more common arguments to `yarn set version` and executing a regexp takes longer than a strict equality check)